volttron/platform/vip/agent/subsystems/ping.py

```diff
@@ -82,6 +82,8 @@ def ping(self, peer, *args):
         result = next(self._results)
         args = list(args)
         args.insert(0, b'ping')
+        if peer == b'pubsub':
+            peer = b''
```

Comment:
@craig8 In that case it's a good idea.

Background in case anyone doesn't know what he's referring to in regards to extra arguments:

The "peer" argument to publish, subscribe, etc. is obsolete now as this change removed the (completely unused) ability to for an agent to have it's own personal pubsub. It's now a no op.